api/src/db/models/opportunity_models.py

```diff
@@ -142,21 +162,20 @@ class OpportunityAssistanceListing(Base, TimestampMixin):
     opportunity_id: Mapped[int] = mapped_column(ForeignKey(Opportunity.opportunity_id), index=True)
     opportunity: Mapped[Opportunity] = relationship(Opportunity)
 
-    program_title: Mapped[str | None]
-
     assistance_listing_number: Mapped[str | None]
+    program_title: Mapped[str | None]
 
     updated_by: Mapped[str | None]
     created_by: Mapped[str | None]
 
 
-class LinkFundingInstrumentOpportunity(Base, TimestampMixin):
-    __tablename__ = "link_funding_instrument_opportunity"
+class LinkFundingInstrumentSummary(Base, TimestampMixin):
+    __tablename__ = "link_funding_instrument_summary"
```

Comment:
Found this name slightly confusing - it feels backwards as the opportunity summary acts like the "parent" and the funding instruments are the child attributes. What do you think of renaming it to `LinkOpportunitySummaryFundingInstrument` / `link_opportunity_summary_funding_instrument`?

Reply:
Sure, I can swap those